modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-impl/src/main/java/com/liferay/portal/workflow/metrics/rest/internal/resource/v1_0/InstanceResourceImpl.java

```diff
@@ -1131,7 +1133,16 @@ private FieldSort _toFieldSort(Sort[] sorts) {
 
 		Sort sort = (Sort)ArrayUtil.getValue(sorts, 0);
 
-		if (StringUtil.equals(sort.getFieldName(), "createDate")) {
+		if (StringUtil.equals(sort.getFieldName(), "assetType")) {
```

Comment:
Hey @MarinhoFeliphe, adapt these changes here to be compatible with the proposed change above(InstanceEntityModel.java:34). Thx